server/config/conditions.ts

```diff
@@ -104,855 +104,906 @@ export default {
       },
     ],
   },
-  additionalConditions: [
```

Comment:
Sorry for the huge diff here, really the only change was adding the `AP` and `PSS` keys to the sets of additional conditions and adding the PSS ones

Reply:
Think I'm going to trust you on this one!